### PR TITLE
Added Tool Tips to each of the toggles

### PR DESCRIPTION
**Pull request by kobryan0619**
Created ToolTips to each of the toggles based on what was in the old dialogue system.  I created new text for these because they needed to modified slightly from the old flow and we want the new and old flow up and running at the same time for A/B testing.

image

## Links

[Part of this larger tech spec](https://docs.google.com/document/d/1AgqK6kVtH1Lhrg7opFKTRKOPOBOC29gF-1Tk4peBak0/edit?usp=sharing)

## Testing story

I tested this on my own machine but did not write any additional tests for this since it seemed like a small change.  I am interested in feedback if NOT writing tests seems like the right call or not.

### CODE REVIEW

File: apps/i18n/common/en_us.json [hunk 1]
```diff
@@ -969,11 +971,14 @@
...
   "explainLessonExtrasLearnMore": "Learn more about Lesson Extras.",
   "explainPairProgramming": "When pair programming is turned on, students can choose to work with a classmate at the same computer. Turn this setting on if you want students to be able to work together while sharing progress.",
   "explainPairProgrammingLearnMore": " Learn more about pair programming.",
   "explainRestrictedSectionEmail": "If set to ‘yes,’ students will not be able to join this section using the section code. You can still add students to the section manually by moving or copying them from another section.",
+  "explainRestrictedSectionEmailToolTip": "If set to ‘yes,’ students will not be able to join this section using the section code. You can still add students to the section manually by moving or copying them from another section.",
```
Comment (bethanyaconnor) [inline apps/i18n/common/en_us.json:981]: Looks like these could be the same string but I like creating a new one for consistency!

File: apps/src/templates/sectionsRefresh/AdvancedSettingToggles.jsx [hunk 2]
```diff
@@ -34,35 +37,83 @@ export default function AdvancedSettingToggles({
...
+          id={'uitest-pair-toggle'}
+          isToggledOn={section.pairingAllowed}
+          onToggle={e => {
+            handlePairProgrammingToggle(e);
+          }}
+          label={i18n.pairProgramming()}
         />
-      )}
-      {assignedUnitLessonExtrasAvailable && (
+        <span data-tip data-for={'pair-toggle-info'}>
```
Comment (bethanyaconnor) [inline apps/src/templates/sectionsRefresh/AdvancedSettingToggles.jsx]: We do have a [HelpTip](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/lib/ui/HelpTip.jsx) component that has a lot of this logic but it's not quite the same -- it looks like we're using a different FA icon. Curious if you think it's worth pulling out the code in this PR into a similar component for reusability?

Comment (kobryan0619) [reply apps/src/templates/sectionsRefresh/AdvancedSettingToggles.jsx]: I didn't know we had that!  I did create a new component and it is MUCH more readable right now, too!

**Review by bethanyaconnor — APPROVED** [review]
🎉 

I was trying to decide if `InfoHelpTip` needed a test. I could go either way -- I'd probably lean towards a very simple "it renders" test that checks for `FontAwesome` and `ReactTooltip` -- but I don't think it's worth blocking this PR for, especially as your OOO soon. Maybe a good followup when you get back?

**Comment by kobryan0619** [comment]
Sounds good @bethanyaconnor!  I will take add a ticket for the test and assign it to myself for next week and then merge this. Thanks!

**Pull request merged**